docs/en/sql-reference/sql-functions/dict-functions/dict_mapping.md

```diff
@@ -0,0 +1,215 @@
+
+    ```SQL
+    CREATE TABLE dest_table1 (
+        id BIGINT,
+        -- This column records the STRING type order number, corresponding to the order_uuid column in the dict table in Example 1.
+        order_uuid STRING, 
+        batch int comment 'used to distinguish different batch loading',
+        -- This column records the BIGINT type order number which mapped with the order_uuid column.
+        -- Because this column is a generated column configured with dict_mapping, the values in this column are automatically obtained from the dict table in Example 1 during data loading.
+        -- Subsequently, this column can be directly used for deduplication and JOIN queries.
+        order_id_int BIGINT AS dict_mapping('dict', order_uuid)
+    )
+    DUPLICATE KEY (id, order_uuid)
+    DISTRIBUTED BY HASH(id);
+    ```
+
+2. When loading simulated data into this table where the `order_id_int` column is configured as `dict_mapping('dict', 'order_uuid')`, StarRocks automatically loads values into the `order_id_int` column based on the mapping relationship between keys and values in the `dict` table.
+
+      ```SQL
+      MySQL [test]> INSERT INTO dest_table1(id, order_uuid, batch) VALUES (1, 'a1', 1), (2, 'a1', 1), (3, 'a3', 1), (4, 'a3', 1);
+      Query OK, 4 rows affected (0.05 sec) 
+      {'label':'insert_e191b9e4-8a98-11ee-b29c-00163e03897d', 'status':'VISIBLE', 'txnId':'72'}
+      
+      MySQL [test]> SELECT * FROM dest_table1;
+      +------+------------+-------+--------------+
+      | id   | order_uuid | batch | order_id_int |
+      +------+------------+-------+--------------+
+      |    1 | a1         |     1 |            1 |
+      |    4 | a3         |     1 |            3 |
+      |    2 | a1         |     1 |            1 |
+      |    3 | a3         |     1 |            3 |
+      +------+------------+-------+--------------+
+      4 rows in set (0.02 sec)
+      ```
+
+    The usage of `dict_mapping` in this example can accelerate [deduplication calculations and JOIN queries](../../../using_starrocks/query_acceleration_with_auto_increment.md). Compared to the previous solutions for building a global dictionary to accelerate precise deduplication, the  solution by using `dict_mapping` is more flexible and user-friendly. Because the mapping values are directly obtained from the dictionary table at the stage "loading mapping relationships between keys and values into the table". You do not need to write statements to join the dictionary table to obtain mapping values. Additionally, this solution supports various data loading methods.<!--For detailed usage, please refer to xxx.-->
+
+**Example 3: If the mapping column in the table is not configured as a generated column, you need to explicitly configure the `dict_mapping` function for the mapping column when loading data into the table, obtain the values mapped to the keys.**
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/27e02e4f-7d11-45fd-a369-e1af143e1000)
-> in order to obtain ...